Parse/Parse.h

```diff
@@ -67,6 +68,22 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)setApplicationId:(NSString *)applicationId clientKey:(NSString *)clientKey;
```

Comment:
My best guess for the reason that we use `set` instead of initialize on iOS - `+initialize` is a method declared by `NSObject`, equivalent to a static contractor in java-land. 

It would have caused some ambiguity for developers if we had a method call like `[Parse initialize:@"appKey" clientKey:@"ck"]`, but we can get around that now by marking `+initialize` as unavailable for use (which wasn't possible when Parse was first created).